### DISCUSSION TITLE
How to specify which binary to run?

### DISCUSSION THREAD

**Discussion by JakenHerman** [cargo-lambda/cargo-lambda#576] [Q&A] [answered]
I have a project that has multiple binaries but one `Cargo.toml` file. Running `cargo lambda watch` boots up the incorrect binary. Is there anything similar to `cargo run --bin <binary_name>` for cargo-lambda? I'm not seeing anything in the discussions, issues, or documentation.

**Comment by calavera** [comment, marked as answer]
You don't need to specify the binary when the emulator starts, you only need to do it when you invoke the function. If there are more than one binary in your project, nothing should build until the first invocation. This might help:

https://www.cargo-lambda.info/commands/invoke.html#multi-function-support
